apps/src/templates/teacherDashboard/SectionActionDropdown.jsx

```diff
@@ -91,16 +93,32 @@ class SectionActionDropdown extends Component {
   };
 
   onClickHideShow = () => {
+    const hideShowEvent = this.props.sectionData.hidden
+      ? EVENTS.SECTION_TABLE_RESTORE_SECTION_CLICKED
+      : EVENTS.SECTION_TABLE_ARCHIVE_SECTION_CLICKED;
+    analyticsReporter.sendEvent(hideShowEvent, {}, PLATFORMS.BOTH);
     this.props.toggleSectionHidden(this.props.sectionData.id);
   };
 
   onClickSync = () => {
+    analyticsReporter.sendEvent(
+      this.props.sectionData.loginType === OAuthSectionTypes.google_classroom
+        ? EVENTS.SECTION_TABLE_SYNC_GOOGLE_CLASSROOM_CLICKED
```

Comment:
This only works if we have only 2 syncs, if there are more they will get lumped into the clever section. Even if we currently only have 2, I would prefer to add some double checking that it is a clever section so that if we do eventually add a third it is counted correctly